mw-webapp/src/utils/auth/writeNewUserCredentialsToDataBase.ts

```diff
@@ -0,0 +1,30 @@
+import {getRedirectResult} from "firebase/auth";
+import {auth} from "src/firebase";
+import {UserService} from "src/service/UserService";
+
+const newUserBlankWays = {
+  ownWays: [""],
+  favoriteWays: [""],
+  mentoringWays: [""],
+};
```

Comment:
transfer

Reply:
1. Change name to writeNewUserCredentials
2. Transfer all User data in one block